README.md

```diff
@@ -27,20 +37,48 @@ be integrated with other applications and languages utilizing its web service.
 * QuickBooks is supported by XBRL-GL.
 
 
-## Installation
+# Installation
 
-Please check out our [Arelle install guide](INSTALLATION.md).
+The implementation is in Python 3.10, and is intended for Windows (any recent),
+Mac OS-X 10.9+, Unix or Linux. The standard installation includes both desktop GUI,
+a localhost RESTful web server and CLI.
 
+## Install PyPi package
+```shell
+pip install arelle-release
```

Comment:
We should document [supported optional extras](https://github.com/Arelle/Arelle/blob/bb7eb53e5d26adbd7fb307d9bb4b1c7a0b5b592d/pyproject.toml#L35-L55):
```suggestion
The Arelle python package defines optional extra dependencies for various plugins and use cases.
* Crypto (security plugin dependencies)
* DB (database plugin dependencies)
* EFM (EdgarRenderer plugin dependencies - does not include the EdgarRenderer, just the dependencies required to run it)
* ObjectMaker (ObjectMaker plugin dependencies)
* WebServer (dependencies for running the Arelle web server)
```shell
pip install arelle-release
# or for all extra dependencies
pip install arelle-release[Crypto,DB,EFM,ObjectMaker,WebServer]
```